packages/modules/viewer-react/src/services/telemetry/Performance.ts

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { trackViewerMetric, viewerAI } from "./TelemetryService";
+
+export type PerformanceMeasures =
+  | "ViewerInitialized"
+  | "BaseViewerInitialized"
+  | "IModelConnected"
+  | "ViewStateCreated"
+  | "TileTreesLoaded";
+
+export type PerformanceMarks =
+  | "ViewerStarting"
+  | "ViewerStarted"
+  | "BaseViewerStarted"
+  | "IModelConnection"
+  | "ViewStateCreation"
+  | "TilesLoaded";
+
+export class ViewerPerformance {
+  private static _enabled: boolean;
+
+  private static _getAiKey(): string {
+    switch (process.env.IMJS_URL_PREFIX) {
+      case "dev-":
+        return "202e4e19-0357-4b93-abb4-52d9c345384d";
+      case "qa-":
+        return "bc9fee8c-d537-4892-b760-750392c531be";
+      default:
+        return "76ebaa63-f57e-4955-aedf-43e2741724ec";
+    }
+  }
+
+  private static async _logMetric(measureName: PerformanceMeasures) {
+    if (!viewerAI.initialized) {
+      await viewerAI.initialize(this._getAiKey());
```

Comment:
you can drop the async and await modifiers, viewer ai init is synchronous

Reply:
fixed